### PR TITLE
File badges

### PR DESCRIPTION
**Pull request by fussel178**
Added nice file badges to video description:
Screenshot_20200718_155647

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,10 +73,16 @@ The `data.json` file for every title has the following format:
...
+  "copyright": "Some nice copyright information to your video",
+  "badges": [
+    "hd-720p",
+    "hevc-h265",
+    "aac-128k"
+  ]
 }
 ```
 All information will be parsed for every title and showed on the website.
+The badges array will be parsed to vector graphics equally named in the `badges` folder.
```
Comment (pklaschka) [inline README.md]: Could you please add a few more details about the naming, here? something like

```suggestion
The `badges` array contains the file names (without extension) that are stored as `[name].svg` in the `badges` directory. `"hd-720p"`, for example, would render the file `badges/hd-720p`.
```

File: index.php [hunk 2]
```diff
@@ -56,6 +57,17 @@ function get_json_from_file($path) {
                     DateTime::createFromFormat(DateTime::ISO8601, $data->release)->format("Y") ?>:
                 <i><?= $data->director ?? "Unbekannter Regisseur" ?></i></p>
             <p><?= $data->description ?? "N/A" ?></p>
+            <div>
+                <?php foreach ($badges as $badge) {
+                    $path = "./badges/$badge.svg";
+                    if (file_exists($path)) {
+                        include $path;
+                    } else {
+                        echo "<!-- Invalid badge: $badge, file not found: $path -->";
```
Comment (pklaschka) [inline index.php]: We need to escape this as well:

```suggestion
                        echo "<!-- Invalid badge: " . htmlspecialchars($badge) . ", file not found: " . htmlspecialchars($path) . " -->";
```

**Review by pklaschka — CHANGES_REQUESTED** [review]
Thank you for the PR. Please see my inline comments.

BTW: It might be useful to call `sd-576p` `pal-576p`, as this is basically the PAL standard. NTSC, on the other hand, would (of course) be `480p`. I'd suggest that we support both formats 😉

**Comment by fussel178** [comment]
> BTW: It might be useful to call `sd-576p` `pal-576p`, as this is basically the PAL standard. NTSC, on the other hand, would (of course) be `480p`. I'd suggest that we support both formats wink

Good call!
I forgot the names for the resolutions.

**Review by pklaschka — APPROVED** [review]
This has been in my code review backlog for too long. Let's merge this for now and if we want, we can improve on that later :see_no_evil:

**Comment by fussel178** [comment]
Well, I tried to create the `pal-576p` and `ntsc-480p` badges. But I currently don't have AdobeXD to open the design file. :see_no_evil:

**Review by pklaschka — COMMENTED** [review]
Let's merge this for now and add the new Icons later :wink:

movefast

**Pull request merged**